eras/conway/impl/src/Cardano/Ledger/Conway/Governance/Procedures.hs

```diff
@@ -162,9 +168,19 @@ data Voter c
 
 instance Crypto c => ToJSON (Voter c)
 
--- TODO: Make it nicer, eg. "DRep-ScriptHash<0xdeadbeef>..."
--- Will also need a ToJSONKey instance for Credential.
-instance Crypto c => ToJSONKey (Voter c)
+instance Crypto c => ToJSONKey (Voter c) where
+  toJSONKey = toJSONKeyText prefixVoter
+    where
+      prefixVoter (CommitteeVoter (KeyHashObj (KeyHash kh))) =
+        "committee-keyhash-" <> hashToTextAsHex kh
```

Comment:
I can't tell if it's possible here to reuse the `ToJSONKey` instance of the credential, which is what I would like to do.
`toJSONKey` returns `ToJSONKeyFunctions` without taking any arguments, so not sure how I would apply this function to the Credential :thinking:

Reply:
Yes you can create a separate function `credentialToText` and reuse it here and in the ToJSONKey instance for Credential